src/components/concept/_ip_addresses.mdx

```diff
@@ -0,0 +1,9 @@
+If you are behind a firewall or using a virtual private network (VPN), you may need to add the following IP addresses to the **ingress** allowlist and **egress** allowlist:
```

Comment:
I might change this to be a little more general, something like "using a virtual private network (VPN), or have network restrictions on access to your SCM or CI provider" - but not required, I think people will pick up fine as it is and we'll have a good place to link to for generic IP questions!